foundations/javascript_basics/fundamentals-5.md

```diff
@@ -28,3 +35,12 @@ At this point you just need a little more practice!  Go back to the [JavaScript
 - findTheOldest
 - (Solutions for these exercises can be found in the 'solutions' branch of that repo)
 </div>
+
+### Knowledge Check
+
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+- <a class="knowledge-check-link" href="https://javascript.info/object#summary">What is the difference between objects and arrays?</a>
+- <a class="knowledge-check-link" href="https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Basics#bracket_notation">How do you access object properties?</a>
+- <a class="knowledge-check-link" href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map">What is _Array.prototype.map()_ useful for?</a>
+- <a class="knowledge-check-link" href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce">What is _Array.prototype.reduce()_ useful for?</a>
```

Comment:
While we do link to MDN in general here we don't link to the array reference in this lesson. I'll have to check with the team but I don't think we want to have knowledge checks that aren't *explicitly* linked to if that makes sense.

Reply:
I can change the link to the video where it was covered. I just thought that a different explanation from the docs would be beneficial.

Reply:
@LeoKirasic yeah we don't want to link to resources that we don't specifically tell learners to review as part of the lesson.

If we're going to link to an external page in the knowledge checks we need to link to that exact same page in the lesson content.